examples/deck/custom-layers/junction-scatterplot-layer.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import { CompositeLayer, ScatterplotLayer } from 'deck.gl';
+
+export default class JunctionScatterplotLayer extends CompositeLayer {
```

Comment:
A better name for this would probably be `StrokedScatterplotLayer` or `OutlinedScatterplotLayer`